vanilla-framework/vanilla-framework
# Update yarn deps

## Done

Update yarn deps in `package.json`

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check no errors
- Check no errors in travis build
